src/main.ts

```diff
@@ -63,22 +64,21 @@ fetchConfig()
     // const authzSerice = new AuthorizationService() // FUTURE
     // await authzSerice.getAuthorizations('NR123456') // FUTURE
     */
-    if (featureFlags.getFlag('bcrs-create-ui-enabled')) {
-      new Vue({
-        vuetify,
-        router,
-        store,
-        render: h => h(App)
-      }).$mount('#app')
-    }
+
+    new Vue({
+      vuetify,
+      router,
+      store,
+      render: h => h(App)
+    }).$mount('#app')
   })
   .catch((error: string) => {
     /**
      * This catches any un-handled errors from fetchConfig()
      * or anything else in then() block above.
      */
     console.error(error) // eslint-disable-line no-console
-    alert('There was an error starting this page. (See console for details.)' +
+    alert('There was an errors starting this page. (See console for details.)' +
```

Comment:
Typo.

Reply:
poor rebase on my part, corrected in latest commit